src/code/views/dropdown-view.coffee

```diff
@@ -46,7 +54,41 @@ module.exports = React.createClass
       )
       (div {className: menuClass, onMouseLeave: @blur, onMouseEnter: @unblur},
         (ul {},
-          (DropdownItem {key: item.name, item: item, select: @select}) for item in @props.items
+          (DropdownItem {name: item.name or item, item: item, select: @select, isActionMenu: @props.isActionMenu}) for item in @props.items
```

Comment:
I think we need to maintain `key` as an attribute, because in some cases (when selecting a palette item for a node I think) we dynamically update this list.  I could be wrong, but I have noticed console warnings about this in the past.  See https://facebook.github.io/react/docs/multiple-components.html#dynamic-children